src/ralph/defaults.py

```diff
@@ -1,141 +1,9 @@
 """Default configurations for Ralph"""
 
-import io
-from enum import Enum
 from os import environ
 from pathlib import Path
 
-import yaml
 from click import get_app_dir
 
-from ralph.exceptions import ConfigurationException
-
-
-class DatabaseBackends(Enum):
-    """Enumerate active database backend modules.
-
-    Adding an entry to this enum will make it available to the CLI.
-    """
-
-    ES = "ralph.backends.database.es.ESDatabase"
-    MONGO = "ralph.backends.database.mongo.MongoDatabase"
-
-
-class Parsers(Enum):
-    """Enumerate active parsers modules.
-
-    Adding an entry to this enum will make it available to the CLI.
-    """
-
-    GELF = "ralph.parsers.GELFParser"
-    ES = "ralph.parsers.ElasticSearchParser"
-
-
-class StorageBackends(Enum):
-    """Enumerate active storage backend modules.
-
-    Adding an entry to this enum will make it available to the CLI.
-    """
-
-    LDP = "ralph.backends.storage.ldp.LDPStorage"
-    FS = "ralph.backends.storage.fs.FSStorage"
-    SWIFT = "ralph.backends.storage.swift.SwiftStorage"
-
-
-class StreamBackends(Enum):
-    """Enumerate streaming backend modules.
-
-    Adding an entry to this enum will make it available to the CLI.
-    """
-
-    WS = "ralph.backends.stream.ws.WSStream"
-
-
-def config(key, default_value):
-    """
-    Get a value based on its key returning the first of (in order):
-        1. Environment
-        2. Config file
-        3. default_value
-    """
-
-    value = environ.get(key, None)
-    if value is not None:
-        return value
-
-    if CONFIG is not None and key in CONFIG:
-        return CONFIG[key]
-
-    return default_value
-
-
-def load_config(config_file_path):
-    """Return a dictionary representing Ralph's configuration."""
-
-    try:
-        with open(config_file_path, encoding=DEFAULT_ENCODING) as config_file:
-            return yaml.safe_load(config_file)
-    except yaml.scanner.ScannerError as exc:
-        raise ConfigurationException("Configuration could not be loaded") from exc
-    except FileNotFoundError:
-        return None
-
-
-DEFAULT_LOGGING_CONFIG = {
-    "version": 1,
-    "propagate": True,
-    "formatters": {
-        "ralph": {"format": "%(asctime)-23s %(levelname)-8s %(name)-8s %(message)s"},
-    },
-    "handlers": {
-        "console": {
-            "class": "logging.StreamHandler",
-            "level": "INFO",
-            "stream": "ext://sys.stderr",
-            "formatter": "ralph",
-        },
-    },
-    "loggers": {
-        "ralph": {
-            "handlers": ["console"],
-            "level": "INFO",
-        },
-        "swiftclient": {
-            "handlers": ["console"],
-            "level": "ERROR",
-        },
-    },
-}
-
 APP_DIR = Path(environ.get("RALPH_APP_DIR", get_app_dir("ralph")))
```

Comment:
Do we keep `APP_DIR` and `MODEL_PATH_SEPARATOR` in defaults.py, or do we move them to conf.py too?

Reply:
Good)